app/controllers/expenses_controller.rb

```diff
@@ -11,12 +11,13 @@ def new
 
   def show
     @expense = Expense.find params[:id]
+    find_expense_type
   end
 
   def create
     find_user
-    expense = @user.expenses.new(expenses_params)
-    if !expense.save
+    @expense = @user.expenses.new(expenses_params)
```

Comment:
i believe chris told us to use "build" vs new when we are creating an object through an association